array_api_tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+from functools import lru_cache
+
+from pytest import mark
+
+from . import _array_module as xp
+from ._array_module import _UndefinedStub
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--ext',
+        '--disable-extension',
```

Comment:
I guess if you add options in a module's conftest.py they don't show up in `pytest -h`.